lib/zold/remotes.rb

```diff
@@ -120,6 +121,7 @@ def initialize(file, network: 'test')
       raise 'Network can\'t be nil' if network.nil?
       @network = network
       @mutex = Mutex.new
+      read_mtime
```

Comment:
@emilianodellacasa have you seen this? https://www.yegor256.com/2015/05/07/ctors-must-be-code-free.html

Reply:
Ok fixed